### PR TITLE
[15.0][ADD] stock_inventory_count_to_zero: New module to set inventory quantities to zero when a recount is requested

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT44404

ping @carlosdauden @chienandalu

### CODE REVIEW

File: stock_request_count_to_zero/models/stock_quant.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+                "name": _("Quantities Already Set"),
+                "type": "ir.actions.act_window",
+                "view_mode": "form",
+                "views": [(view.id, "form")],
+                "view_id": view.id,
+                "res_model": "stock.inventory.warning",
+                "target": "new",
+                "context": ctx,
+            }
+        for quant in self:
```
Comment (rousseldenis) [inline stock_request_count_to_zero/models/stock_quant.py]:
```suggestion
        self.update({
            "inventory_quantity": 0.0,
            "user_id": self.env.user.id,
            "inventory_quantity_set": True,
        })
```

**Review by rousseldenis — APPROVED** [review]
Minor comment

**Comment by pedrobaeza** [comment]
As there a set of modules called `stock_request`, this module name may be misleading. Better to call it `stock_inventory_count_to_zero` or `stock_quant_count_to_zero`.

**Comment by rousseldenis** [comment]
> As there a set of module called `stock_request`, this module name may be misleading. Better to call it `stock_inventory_count_to_zero` or `stock_quant_count_to_zero`.

I wouldn't say better :+1:

**Review by chienandalu — APPROVED** [review]
Aside from module renaming, functionally ok :+1:

**Comment by sergio-teruel** [comment]
@pedrobaeza @rousseldenis @chienandalu  Changes done!!! I have changed the name to stock_inventory_count_to_zero

**Review by chienandalu — APPROVED** [review]

**Pull request merged**